source/slang/slang-ir-insts.h

```diff
@@ -1018,6 +1019,17 @@ struct IRBackwardDifferentiate : IRInst
     IR_LEAF_ISA(BackwardDifferentiate)
 };
 
+struct IRIsDifferentialNull : IRInst
```

Comment:
You might be able to use `IRIsType` inst here, but I see this is also a simpler use case.